OneTimePasswordLegacy/OTPToken.swift

```diff
@@ -119,13 +119,13 @@ public extension OTPToken {
             return false
         }
         if let keychainItem = self.keychainItem {
-            guard let newKeychainItem = updateKeychainItem(keychainItem, withToken: token) else {
+            guard let newKeychainItem = Keychain.sharedInstance.updatePersistentToken(keychainItem, withToken: token) else {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 124 characters